modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc

```diff
@@ -435,7 +435,7 @@ By default, when a user loses access to a channel, the next Couchbase Lite Pull
 
 Documents that exist in multiple channels belonging to the user (even if they are not actively replicating that channel) are not auto-purged unless the user loses access to all channels.
 
-Users will be receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation; this is sent regardless of the current auto-purge setting.
+Users will receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation when the auto-purge setting is set to `true`.
```

Comment:
There's an ambiguity here: is it

1. "Users will receive (x if y) when z", or
2. "Users will receive x if (y when z)"

Do you see what I mean? I'm guessing it's the first one. To avoid this ambiguity, I would rephrase as

"When z, users will receive x if y."

```suggestion
When the auto-purge setting is set to `true`, users will receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation.
```